### DISCUSSION TITLE
Manually change item selection and pass additional data to items?

### DISCUSSION THREAD

**Discussion by michaelbdavid** [prabhuignoto/react-chrono#230] [Q&A]
Hello,

Just starting out using chrono (its awesome!) and I'm trying to find a way that I can change item selection dynamically.  So I'm trying to sync up video play with chrono so that at certain points of the video I would switch items.  

Also, I would like to pass additional metadata (date/time) to each item is there a way to do these?

One more thing is there a way to hide the cards if we do not want them to display?

Thanks for the help!

**Comment by prabhuignoto** [comment]
Hey @michaelbdavid glad you liked it.

Are you asking how to select a timeline item without having to interact with the timeline?

>Also, I would like to pass additional metadata (date/time) to each item is there a way to do these?

Right now there is no way to do this. I will add this to our roadmap and will try to get this pretty soon.

>One more thing is there a way to hide the cards if we do not want them to display?

Again this is not possible at the moment but this will be included in the release planned for this weekend.

Hope, I answered your questions. Please feel free add more context if you like.

**Reply by michaelbdavid** [reply]
Hey @prabhuignoto
> 
> Are you asking how to select a timeline item without having to interact with the timeline?

Correct I'd like to update the timeline to an item that matches current time in a video.  So for example lets say I have a bird at time 9.5 seconds I would like to move to the item for bird.  Even if a user is currently not interacting with chrono.  So when they do look at chrono is at the correct item within the timeline, otherwise chrono remains at the first time.

It would be nice if there was a way to have timeline items also have a date/time stamp and display themselves accordingly.  

Another question that I had is there a way to adjust how far apart the timeline items are spaced?  I think this would go along with adding date/time stamp support as closer items would display closer together while further apart items would be further.

Thanks again for the reply and answering my questions,
Michael

**Reply by ansonjhuang** [reply]
I would love this feature! I'm using Chrono for a YouTube Chrome extension project and this would be incredibly helpful.
